### ISSUE TITLE
Images Can't Be Read by Screen Readers

### ISSUE THREAD

**Issue by bbertucc** [WordPress/five-for-the-future#326]
## Problem
There are many images with missing alt text, meaning screen readers can't read the images.

For instance, every headshot is missing alt text on https://wordpress.org/five-for-the-future/testimonials/. 

Here is a complete list of images with missing text for your review:  [missing-alt.csv](https://github.com/user-attachments/files/17135516/missing-alt.csv)

**Comment by ryelle** [comment]
Thanks for the issue! It would be helpful to have some more details for each page/case, since they're not all equal. Here's the list from the CSV, with some notes and questions.

- [Homepage](https://wordpress.org/five-for-the-future/)
    All images have empty alt, but some actually _are_ decorative (ex, the icons). I can see adding alt text to the two group photos & maybe headshots.
- [Testimonials](https://wordpress.org/five-for-the-future/testimonials/)
    The headshots are missing alt text, but would "Photo of [Name]" be acceptable/useful, since the name is right next to it?
- [My pledges](https://wordpress.org/five-for-the-future/my-pledges/)
    This is the current user's avatar (or a placeholder if logged out). Kind of same question as above, is "Your avatar" really a helpful alt text?
- [For individuals page](https://wordpress.org/five-for-the-future/for-individuals/)
    The steps screenshotted here are described in text, so does this need alt text?
- [Handbook: Onboarding](https://wordpress.org/five-for-the-future/handbook/organization-pledge-guide/onboarding-organization-and-team-setup/)
    Same as above, these screenshots are just illustrating the text
- [Handbook: About](https://wordpress.org/five-for-the-future/handbook/about-five-for-the-future/)
    This image should probably just be text. If having a shareable infographic is important it could be additional on the page.

Once it's determined which images actually need alt text and what alt text would be useful, I can see that those changes are made.

**Comment by bbertucc** [comment]
Tagging @joedolson on this one. I'm not quite sure of WordPress alt text policies myself.

**Comment by joedolson** [comment]
Homepage: Icons and testimonial headshots should stay empty; adding descriptive text for the photos would be good.

Testimonials: Leave headshots empty.

My pledges: I think it could be beneficial for a user to know that their gravatar is present on this page, so add alt text here. It's not duplicate text like it would be in testimonials.

For individuals: No, those steps are described in detail. Alt text would be redundant.

Handbook: Onboarding: These are a bit less directly related to the text, so maybe, but I wouldn't consider it necessary. 

(Aside: I don't think the 3rd bullet point under "How to resent the confirmation email..." is right. "Click the Resend Confirmation button near the contributor you want to remove.". You're not trying to remove a contributor at this point.)

About: Agreed; this image should just be converted into text.
